src/content/docs/release-notes/mobile-release-notes/ios-release-notes/xcframework-agent-700.mdx

```diff
@@ -1,5 +1,6 @@
 ---
-subject: XCFramework agent
+subject: iOS agent
+title: XCFramework agent
```

Comment:
Should this also be replaced by `iOS agent`? I have the same question for the rest of the release notes.

Reply:
The doc eng team made this change so that the title would still remain historically XCFramework (and tvOS for the other docs) but would be under the iOS umbrella via the subject.